_includes/homepage/intro.html

```diff
@@ -3,7 +3,7 @@
   <div class="container">
     <div class="intro__inner">
       <div class="intro__copy">
-        <h2 class="intro__title">We take it personally</h2>
+        <h2 class="intro__title">We take it personally!</h2>
```

Comment:
@SerbanCiprian, the exclamation was added as a test? Looks like it's outside the scope of the work.

Reply:
Yeah, sorry, was added as a test. I forgot to change.

Reply:
Ok, no problem. Please change it back in your next PR. Thanks! Enjoy you evening :)